app/src/main/java/se/inera/intyg/minaintyg/certificate/dto/CertificateRequestDTO.java

```diff
@@ -0,0 +1,15 @@
+package se.inera.intyg.minaintyg.certificate.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class CertificateRequestDTO {
```

Comment:
Is this really used?

Reply:
It's used in some files I set up in the beginning before we split up the jira. I thought they could be there for now and be removed/changed when its integrated with the other part of the solution.